### DISCUSSION TITLE
S7 error artisan and using phidgeds

### DISCUSSION THREAD

**Discussion by keikoffie** [artisan-roaster-scope/artisan#645] [Q&A]
Hi, i have a error S7 connection. Just installed artisan. also installed phidgeds and i can see tc temperatures on there program. But on Artisan i het the error. 

It's a FIME italian coffee roaster 35kg. Just Total refurbished and want to run on Artisan. Perhaps someone can help me? 

Phidgeds tc 2x hub and hub to connect with USB kabel to PC. 

All help is welcome want to use the artisan instead of small phidgeds screens. 

Greetings 
Maurice from Netherlands.

**Comment by keikoffie** [comment]
![Uploading IMG_20210721_112715048.jpg…]()

**Reply by poundy** [reply]
this image didn't appear to be loaded to Github properly.

You posted in the giesen thread, Lets keep your (completely different) problem over here instead.

**Reply by keikoffie** [reply]
Sorry you are right. I have set the factory settings and the error is gone. But now there is scope monitoring... And no et and bt. What is the problem you think?

**Reply by poundy** [reply]
its your phidgets setup. If you reset to factory, did you then go and configure anything?? 

Attach your settings file would help others clearly see what you have set.

And please stop posting in that other thread......

**Reply by keikoffie** [reply]
Yes i have done the factory settings, after that on config: machine > phidgeds > vint tmp 1101 2x tc

**Reply by keikoffie** [reply]
Must i do something on the phidgeds control panel? On the phidgeds control panel i can Read the temperature when info to port and select the thermocouple 0 and 1.

**Reply by poundy** [reply]
ATTACH YOUR SETTINGS. Can I make that any clearer ? :) 

The issue is likely how you installed (or did not) install the Phidgets driver. You'll need to list your OS and how you installed the driver.

**Comment by keikoffie** [comment]
![Uploading IMG_20210722_140543969.jpg…]()

**Reply by keikoffie** [reply]
I think i can't add a picture

**Reply by poundy** [reply]
Let me try: 
![image](https://user-images.githubusercontent.com/279966/126726294-d0e87166-1b53-4cf2-8042-5918581f60ef.png)

**Reply by poundy** [reply]
yep, it's working fine. Just you.

**Reply by MAKOMO** [reply]
You might just answer emails with images attached instead of browsing to this discussion board on https://github.com/artisan-roaster-scope/artisan/discussions/645 and typing in your messages and add images via drag-and-drop.

**Reply by keikoffie** [reply]
Hi this morning i started the pc and its working:-)

**Reply by keikoffie** [reply]
Many thanks for helping!
